internal/cmd/keys.go

```diff
@@ -219,6 +221,6 @@ func newKeysListCmd(cli *CLI) *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&options.UserName, "user", "", "The name of a user to list access keys for")
-
+	cmd.Flags().BoolVar(&options.ShowExpired, "show-expired", false, "Show expired access keys")
```

Comment:
I would call this `--all` instead since it'll list both expired and unexpired grants

Reply:
The risk with `--all` is that in the future if we have other filters it's less obvious what `--all` should do. Should it change to show all the things hidden by default?

Something like `--show-expired` is nice because the behaviour should not need to change even when we default hide other keys.

Reply:
I think I'm going to go with show-expired because it's the most detailed/accurate. Although I'm also a fan of one word flags, this keeps it consistent with the API and we already have some multi-word flags.